rbd/mirror.go

```diff
@@ -71,6 +71,28 @@ func (imm ImageMirrorMode) String() string {
 	}
 }
 
+//  GetMirrorUuid returns the mirroring uuid for the pool associated with the ioctx.
+//
+//  Implements:
+//  int rbd_mirror_uuid_get(rados_ioctx_t io_ctx,
+//	                       char *uuid, size_t *max_len);
+func GetMirrorUuid(ioctx *rados.IOContext) (string, error) {
```

Comment:
This should probably be `GetMirrorUUID` by Go convention (IIRC).

Reply:
I was following the CamelCase naming convention, but it seems like `UUID` has to be in all caps.

Reply:
No worries. Certain terms are not really words, like "ID" and "UUID" so the letters after the first remain uppercased, IIUC. I also forget the conventions myself from time to time, so it's no biggie. :-)